### ISSUE TITLE
Calling .bootstrapWizard('previous') does not seem to trigger onTabShow event.

### ISSUE THREAD

**Issue by AronDavis** [VinceG/twitter-bootstrap-wizard#185]

**Comment by jiggneshhgohel** [comment]
@AronDavis Did you found a solution to this? I am using version **version 1.3.1**

**Comment by AronDavis** [comment]
@jiggneshhgohel I don't remember if I did, but you can probably rig something with jQuery to have it trigger the event.

**Comment by jiggneshhgohel** [comment]
Thanks @AronDavis for your prompt response and suggestion. I will try everything I can because this behaviour is hampering one of the critical features in my application.

@VinceG if you can be kind enough to comment along it would be a great help.

Thanks.

**Comment by VinceG** [comment]
@jiggneshhgohel Yes it won't call on tab show when you use previous/next events. You can call reset which will trigger the ontabshow but will also detach and reattach the handlers.

**Comment by jiggneshhgohel** [comment]
Thanks @VinceG. I will try your suggested solution.

**Comment by jiggneshhgohel** [comment]
I solved my problem using the combination of `onNext` and `onPrevious` and `OnTabShow` events callback as demonstrated in the examples https://github.com/VinceG/twitter-bootstrap-wizard/blob/1.3.1/examples/basic-formvalidation.html#L157

I added 1.3.1 version examples link because I am using 1.3.1 version in my application.

Thanks for all the support.
